packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page.tsx

```diff
@@ -132,6 +139,41 @@ const Page: NextPageWithLayout = () => {
 
   const entityType = remoteEntityType ?? draftEntityType;
 
+  useEffect(() => {
+    if (authenticatedUser && !namespace) {
+      void router.replace(
+        `/@${authenticatedUser.shortname}/types/new/entity-type`,
+      );
+    }
+  }, [authenticatedUser, namespace, router]);
+
+  useEffect(() => {
+    if (authenticatedUser && !namespace) {
+      // eslint-disable-next-line no-console
+      console.error(
```

Comment:
Feels more like a warning

Reply:
Switched to console.warn in 5d8cf7172392b7df3f295be1c44481f3e34b8e10.